Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
It works for me to just have it as a dependency without reinstalling, so I'm not sure what's happening for you on this.